MainApp/public/app/controllers/authCtrl.js

```diff
@@ -82,7 +83,17 @@ angular.module('authControllers', ['authServices'])
             });
         };
 
-
+        let getClientId=function(){
+            var metaTags=document.getElementsByTagName('meta');
```

Comment:
Does this fix [the meta tag issue](https://github.com/yousuf031991/HumanAgentInteraction/issues/25) ?